tutorials/3d/procedural_geometry/index.rst

```diff
@@ -31,21 +31,21 @@ What is a Mesh?
 ---------------
 
 Many things in Godot have mesh in their name: the :ref:`Mesh <class_Mesh>`, the :ref:`ArrayMesh <class_ArrayMesh>`,
-the :ref:`MeshInstance <class_MeshInstance>`, the :ref:`MultiMesh <class_MultiMesh>`, and
-the :ref:`MultiMeshInstance <class_MultiMeshInstance>`. While they are all related, they have slightly different uses.
+the :ref:`MeshInstance3D <class_MeshInstance3D>`, the :ref:`MultiMesh <class_MultiMesh>`, and
+the :ref:`MultiMeshInstance3D <class_MultiMeshInstance3D>`. While they are all related, they have slightly different uses.
 
-Meshes and ArrayMeshes are resources that are drawn using a MeshInstance node. Resources like
-Meshes and ArrayMeshes cannot be added to the scene directly. A MeshInstance represents one
-instance of a mesh in your scene. You can reuse a single mesh in multiple MeshInstances
+Meshes and ArrayMeshes are resources that are drawn using a MeshInstance3D node. Resources like
+Meshes and ArrayMeshes cannot be added to the scene directly. A MeshInstance3D represents one
+instance of a mesh in your scene. You can reuse a single mesh in multiple MeshInstance3Ds
 to draw it in different parts of your scene with different materials or transformations (scale,
 rotation, position etc.).
 
 If you are going to draw the same object many times, it can be helpful to use a MultiMesh with
-a MultiMeshInstance. The MultiMeshInstance draws meshes thousands of times very
-cheaply. It takes advantage of hardware instancing in order to do so. The drawback with
-using a MultiMeshInstance is that you are limited to one material for all instances. It uses an
-instance array to store different colors and transformations for each instance, but all the
-instances use the same material.
+a MultiMeshInstance3D. MultiMeshInstance3Ds draws meshes thousands of times very
+cheaply by taking advantage of hardware instancing. The drawback with
+using a MultiMeshInstance3D is that each of your mesh's surfaces are limited to one material for
+all instances. It uses an instance array to store different colors and transformations for each
+instance, but all the instances use the same materials.
```

Comment:
I might have misunderstood the terminology - can't an instance have multiple surfaces, which use different materials?

![image](https://user-images.githubusercontent.com/61291296/180598232-1c56f9ed-38a3-4459-9850-443cf8d66815.png)

Reply:
Ah yes, that is confusing. Perhaps rephrasing would be better. The way I read it we are referring to a single surface (because the previous sentence says "each of your mesh's surfaces are limited to one material for all instances") But I guess in the sentence it is ambiguous if we are talking about a all the instances of a surface or all the instances of all the surfaces. 

Maybe change "but all the instances use the same materials" to "but all the instances of each surface use the same material"